java/src/jmri/jmrit/logixng/util/parser/functions/FunctionsBundle.properties

```diff
@@ -200,6 +200,40 @@ the brightness be between 20 and 100. You can then use the following formula:
 </html>
 
 
+NamedBean.getLogixNGTable_Descr     = <html>                                                \
+<h1>getLogixNGTable()</h1>                                                                  \
+<h2>getLogixNGTable(name)</h2>                                                              \
+Returns the LogixNG table <i>name</i>. If the table doesn't exist, null is                  \
+returned.                                                                                   \
+<p>                                                                                         \
+Example:                                                                                    \
+<p>                                                                                         \
+<b>getLogixNGTable("MyTable")</b>                                                           \
+<p>                                                                                         \
+Returns the LogixNG table "MyTable".                                                        \
+<p>                                                                                         \
+<b>table = getLogixNGTable("MyTable")</b><br>                                               \
+Assigns the table "MyTable" to the local variable "table".                                  \
+<p>                                                                                         \
+<b>numColumns = table.numColumns()</b>                                                      \
+Assigns the number of columns of the table in the local variable "table"                    \
+to the local variable "numColumns".                                                         \
+<p>                                                                                         \
+<b>numRows = table.numRows()</b>                                                            \
+Assigns the number of rows of the table in the local variable "table"                       \
+to the local variable "numRows".                                                            \
+<p>                                                                                         \
+<b>cell = table.getCell(row, column)</b>                                                    \
+Assigns the value of a cell of the table in the local variable "table"                      \
+to the local variable "cell".                                                               \
+<p>                                                                                         \
+<b>table.setCell(value, row, column)</b>                                                    \
+Sets the value of a cell of the table in the local variable "table"                         \
+from the local variable "cell".                                                             \
```

Comment:
This needs to say something like "from a local variable containing the new cell value."  

Can the "value" param also be a number or a string?

Reply:
> This needs to say something like "from a local variable containing the new cell value."

I have updated the text now. Note that `value` doesn't need to be a local variable. That was a misstake from me. This works fine:

`table.setCell("Daniel: " + cell, row, column)`

> Can the "value" param also be a number or a string?

Yes. LogixNG tables uses `Object` for cell values internally. So any data type is allowed. But since tables are created by CSV files, its values are strings since each cell in the CSV file is stored in the table as a string.

Reply:
I noticed that I have forgot some `<br>` in this description. They are added now.